src/com/hms/flexyosisoftconnector/DataPoster.java

```diff
@@ -4,7 +4,7 @@
 
 public class DataPoster extends Thread{
 
-   private static final int maxDataPointsPerPost = 200;
+   private int maxDataPointsPerPost = 200;

```

Comment:
This might be more clear if it was like 

```
private static final int MAX_DATA_POINTS_PER_POST_API    = 200;
private static final int MAX_DATA_POINTS_PER_POST_OMF = 5000;

private int maxDataPointsPerPost;
```

Then set them both when things are initialized.